static/javascript/components/bar-charts/group-vertical-bar-chart.js

```diff
@@ -19,18 +18,16 @@ const DATA = [
 ];
 
 class SvgVerticalGroupedBarChart extends React.Component {
```

Comment:
object level properties works fine, is there a reason you removed them?

Reply:
there was an eslint error when I did `npm start`